criu/kerndat.c

```diff
@@ -1720,6 +1720,21 @@ static int kerndat_has_close_range(void)
 	return 0;
 }
 
+static int kerndat_has_timer_cr_ids(void)
+{
+	if (prctl(PR_TIMER_CREATE_RESTORE_IDS, PR_TIMER_CREATE_RESTORE_IDS_OFF, 0, 0, 0) == -1) {
```

Comment:
Please use `PR_TIMER_CREATE_RESTORE_IDS_GET` instead, which is more safe than disabling.